.eslintignore

```diff
@@ -40,6 +40,7 @@ projects/plugins/boost/app/assets/dist/**
 projects/plugins/boost/tests/**
 projects/plugins/boost/docker/**
 projects/plugins/boost/wordpress/**
+!projects/plugins/boost/app/assets/src/js/*.d.ts
```

Comment:
I see there are two .d.ts files in the repo, `projects/plugins/boost/app/assets/src/js/global.d.ts` and `projects/plugins/boost/app/assets/src/js/svg/svgs.d.ts`. This rule only matches the first.

If you want to match both, you could do like
```suggestion
!projects/plugins/boost/app/assets/src/js/**/*.d.ts
```

And similarly in your local .eslintignore.

Reply:
Yep, tested and the more globby path also catches svgs.d.ts. Nice catch! Committed in 91db7f771

Reply:
Thank you @anomiex for taking a look at the PR and spotting this one 🙏🏻
Also thank you @thingalon 👍 

I have added the globby path to the local .eslintignore file too in commit 581e22441cb4c15b98c3b4fff78a7e2ece6cdcf0